ci/Publish-CalendarMobileArtifacts.Jenkinsfile

```diff
@@ -0,0 +1,221 @@
+			description: "Generate Release notes for this build."
+		 )
+    }
+
+    agent any
+
+    stages {
+    	stage("Checking params") {
+			steps {
+				script{
+					if(!params.googlePlayStore && !params.appleAppStore) {
+						currentBuild.result = 'ABORTED'
+						error('No artifacts were selected.')
+					}
+				}
+				echo "Params OKAY"
+			}
+    	}
+		stage("Prepare Release Notes") {
+			environment {
+				VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+			}
+			when {
+				expression {
+					params.generateReleaseNotes && (params.googlePlayStore || params.appleAppStore)
+				}
+			}
+			steps {
+				sh "npm ci"
+				script { // create release notes
+					def android = params.googlePlayStore ? pregenerateReleaseNotes("android", env.VERSION) : null
+					def ios = params.appleAppStore ? pregenerateReleaseNotes("ios", env.VERSION) : null
+
+					// Assigns the dict returned by reviewReleaseNotes with the notes for each platform to the global var releaseNotes
+					releaseNotes = reviewReleaseNotes(android, ios, env.VERSION)
+					echo("${releaseNotes}")
+				}
+			} // steps
+		} // stage Prepare Release Notes
+		stage("Publishing Artifacts") {
+			parallel {
+				stage("Android App") {
+					environment {
+						VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+						FILE_PATH = "build/app-android/calendar-tutao-release-${VERSION}.apk"
+						GITHUB_RELEASE_PAGE = "https://github.com/tutao/tutanota/releases/tag/tuta-calendar-android-release-${VERSION}"
+					}
+					when {
+						expression {
+							params.googlePlayStore
+						}
+					}
+					steps {
+						script {
+							def util = load "ci/jenkins-lib/util.groovy"
+							util.downloadFromNexus(	groupId: "app",
+													artifactId: "calendar-android",
+													version: "${env.VERSION}",
+													outFile: "${env.WORKSPACE}/${env.FILE_PATH}",
+													fileExtension: 'apk')
+							if (!fileExists("${env.FILE_PATH}")) {
+								currentBuild.result = 'ABORTED'
+								error("Unable to find file ${env.FILE_PATH}")
+							}
+							echo "File ${env.FILE_PATH} found!"
+
+							catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS', message: 'Failed to upload android test app to Play Store') {
+								androidApkUpload(
+									googleCredentialsId: 'android-app-publisher-credentials',
+									apkFilesPattern: "${env.FILE_PATH}",
+									trackName: 'production',
+									// Don't publish the app to users directly
+									// It will require manual intervention at play.google.com/console
+									rolloutPercentage: '0%',
+									recentChangeList: [
+										[
+											language: "en-US",
+											text    : "see: ${env.GITHUB_RELEASE_PAGE}"
+										]
+									]
+								)
+							}
+						} // script
+					} // steps
+				} // stage Android App
+				stage("iOS App") {
+					environment {
+						VERSION = "${params.appVersion.trim() ?: env.VERSION}"
+						FILE_PATH = "build/app-ios/calendar-tutao-${VERSION}-release.ipa"
```

Comment:
same here, it doesn't match the file path I see on nexus or defined in the build script. But I think there is something I'm missing, so I may be wrong